.evergreen/config.yml

```diff
@@ -17,6 +17,12 @@ command_type: system
 # That roughly accounts for variable system performance for various buildvariants
 exec_timeout_secs: 1800 # 6 minutes is the longest we'll ever run
 
+# don't schedule tests if a commit only changes non-code files 
+ignore:
+  - "*.rst" 
+  - "LICENSE"
+  - "THIRD-PARTY-NOTICES"
```

Comment:
Can you also add the changelog: "doc/changelog.rst". I don't think we'd ever put doctests in there and it often gets updated by itself.